cfgov/ask_cfpb/scripts/export_ask_data.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import unicode_literals
+
+import datetime
+
+import HTMLParser
+from django.utils import html
+
+from paying_for_college.csvkit import csvkit
```

Comment:
agate (and its predecessor csvkit) is a heavy package with lots of dependencies, some of which can have version clashes with our cfgov requirements. So I couldn't justify including it.
PFC borrowed the unicode-aware readers and writers from csvkit, which are great line cooks for unicode sandwiches.
If we ever get to python 3, this issue goes away.

I'll trap for existence of paying_for_college in INSTALLED_APPS.